src/locales/en.json

```
@@ -340,7 +340,9 @@
       "SortingHighestRated": "Highest Rated",
       "SortingMostPopular": "Most Popular",
       "Clear": "Clear",
-      "CloseSearch": "Close Search"
+      "CloseSearch": "Close Search",
+      "ShowFiltered": "Show filtered content",
+      "HideFiltered": "Hide filtered content"
```

I think the best is to use the same text for both states
```suggestion
      "ShowFiltered": "Filter content",
      "HideFiltered": "Filter content"
```
or may be "Safe content only" could also be a good title